doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -193,7 +194,15 @@ See :ref:`cmake_options` for more information about defining CMake options.
 Performing prerequisite operations
 **********************************
 
```

Comment:
You need to also write one additional sentence before creating subsections. E.g.

`To support the Google Fast Pair subsystem in your project, you need to follow the steps in the following subsections that describe necessary Kconfig and API actions.`